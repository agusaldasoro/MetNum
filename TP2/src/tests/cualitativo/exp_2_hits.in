1 -1 0 tests/cualitativo/exp_2.txt 0.00001