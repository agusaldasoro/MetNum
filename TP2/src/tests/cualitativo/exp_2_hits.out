0.197960
0.526694
0.595271
0.351336
0.108450
0.295932
0.144997
0.000000
0.144997
0.000000
0.000000
0.253447
0.000000
0.000000
0.000000
0.000000
0.398297
0.536352
0.195904
0.000000
0.383318
0.401151
0.160928
0.160928
0.160928
0.142386
0.142386
0.142386
0.142386
0.094983
0.094983
0.174987
